services/webnn/ort/graph_builder_ort.cc

```diff
@@ -673,15 +703,12 @@ GraphBuilderOrt::AddInstanceNormalizationOperation(
     std::vector<float> scale_data(input_channel, 1.0f);
     switch (input_data_type) {
       case OperandDataType::kFloat16: {
-        std::vector<uint16_t> scale_data_fp16 =
-            ConvertFloat32ToFloat16(scale_data);
-        scale_name = CreateInitializer<uint16_t>(constant_dims, scale_data_fp16,
-                                                 input_data_type);
+        std::vector<Float16> scale_data_fp16 = Float16FromFloat32(scale_data);
```

Comment:
Construct `scale_data_fp16` directly? I know this is an old issue, would be great to fix in this PR: 
```suggestion
        std::vector<Float16> scale_data_fp16(input_channel, Float16FromFloat32(1.0f));
```